ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchaserInfoHybridAdditionsTests.swift

```diff
@@ -8,95 +8,97 @@
                     expect(dictionary["managementURL"] as? NSNull) == NSNull()
                 }
             }
             context("nonSubscriptionTransactions") {
                 it("contains all the non subscription transactions") {
-                    let purchaserInfo = PartialMockPurchaserInfo()
-                    
-                    let transactionDate = Date()
-                    let transaction = PurchasesCoreSwift.Transaction(transactionId: "transactionid", productId: "productid", purchaseDate: transactionDate as Date)
-                    purchaserInfo.stubbedNonSubscriptionTransactions = [transaction]
-                    
-                    let dictionary = purchaserInfo.dictionary
+                    let transactionDateString = "1990-08-30T02:40:36Z"
+
+                    let mockCustomerInfo = try CustomerInfo(data: [
+                        "request_date": "2019-08-16T10:30:42Z",
+                        "subscriber": [
+                            "original_app_user_id": "app_user_id",
+                            "first_seen": "2019-06-17T16:05:33Z",
+                            "subscriptions": [:],
+                            "non_subscriptions": [
+                                "productid": [
+                                    [
+                                        "id": "transactionid",
+                                        "is_sandbox": true,
+                                        "original_purchase_date": "1990-08-30T02:40:36Z",
+                                        "purchase_date": transactionDateString,
+                                        "store": "play_store"
+                                    ]
+                                ]
+                            ]
+                        ]])
+
+                    let dateformatter = ISO8601DateFormatter()
+                    let transactionDate = dateformatter.date(from: transactionDateString)!
+
+                    let dictionary = mockCustomerInfo.dictionary
                     let nonSubscriptionTransactions = dictionary["nonSubscriptionTransactions"] as? Array<Any>
                     expect(nonSubscriptionTransactions?.count) == 1
-                    
+
                     let transactionDictionary = nonSubscriptionTransactions?[0] as? Dictionary<String, Any>
-                    expect(transactionDictionary?["revenueCatId"] as? String) == transaction.revenueCatId
-                    expect(transactionDictionary?["productId"] as? String) == transaction.productId
-                    expect(transactionDictionary?["purchaseDateMillis"] as? Double) == (transactionDate as NSDate).rc_millisecondsSince1970AsDouble()
-                    
-                    let dateformatter = ISO8601DateFormatter()
+                    expect(transactionDictionary?["revenueCatId"] as? String) == "transactionid"
+                    expect(transactionDictionary?["productId"] as? String) == "productid"
```

Comment:
I would extract both `"productid"` and `"transactionid"` to something like `expectedTransactionId` and `expectedProductId`

Reply:
sounds good! I'll do that in the last PR though so that I can merge them sequentially

Reply:
done!

Reply:
https://github.com/RevenueCat/purchases-hybrid-common/pull/133/commits/1f3991065d0278d795522ee031adc365369f9c67